src/uiprotect/websocket.py

```diff
@@ -117,7 +119,7 @@ def _state_changed(self, state: WebsocketState) -> None:
     async def _websocket_inner_loop(self, url: URL) -> None:
         _LOGGER.debug("Connecting WS to %s", url)
         await self._attempt_auth(False)
-        ssl = None if self.verify else False
+        ssl = True if self.verify else False
```

Comment:
I feel like this can be simplified now? 😄

Reply:
Yes it can. I fixed the typing and didn’t think about it otherwise

Reply:
https://github.com/uilibs/uiprotect/pull/175